com.unity.render-pipelines.core/Runtime/Common/DynamicResolutionHandler.cs

```diff
@@ -501,6 +501,10 @@ public Vector2Int GetScaledSize(Vector2Int size)
             }
 
             Vector2Int scaledSize = ApplyScalesOnSize(size);
+
+            scaledSize.x = Math.Min(scaledSize.x, size.x);
```

Comment:
I think we should do this in ApplyScalesOnSize though as that is public API too, specifically this variant :

> Vector2Int ApplyScalesOnSize(Vector2Int size, Vector2 scales)